courses/free-courses-en.md

```diff
@@ -200,6 +200,7 @@
 * [Android Basics: User Interface](https://www.udacity.com/course/android-basics-user-interface--ud834) (Udacity)
 * [Android Developer Fundamentals (Version 2) — Codelab](https://developer.android.com/courses/fundamentals-training/toc-v2)
 * [Android Developer Fundamentals (Version 2) — Concepts](https://google-developer-training.github.io/android-developer-fundamentals-course-concepts-v2/index.html)
+* [Android Development for Beginners - Full Course](https://www.youtube.com/watch?v=fis26HvvDII) -  MeiCode(freeCodeCamp)
```

Comment:
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

Reply:
@eshellman This is a video of 11+ hrs which has a duration more than that of many course playlists. Moreover, this course is a full course in one video. So, I believe it serves the purpose and can be included in the repo.